python/apikey/client.py

```diff
@@ -103,15 +108,16 @@ def get_document(self, did):
 
         return self._api.request('get', '/api/documents/' + did)
 
-    def list_documents(self):
+    def list_documents(self, query={}):
         '''
         Get list of documents for current user.
 
         Returns:
             - requests.Response: Onshape response data
         '''
 
-        return self._api.request('get', '/api/documents')
+        # return self._api.request('get', '/api/documents')
```

Comment:
I apologize if this commented out line was in earlier pull requests and I missed it, but it needs to be deleted too.